### PR TITLE
Added expo app example

### PR DESCRIPTION
**Pull request by Manitej66**
Added expo app example

### CODE REVIEW

File: _examples/how-to-create-a-flutter-app-with-serverless.md [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+### Adding the table
+
+We'll be using [Amazon DynamoDB](https://aws.amazon.com/dynamodb/); a reliable and highly-performant NoSQL database that can be configured as a true serverless database. Meaning that it'll scale up and down automatically. And you won't get charged if you are not using it.
+
+{%change%} Replace the `stacks/MyStack.js` with the following.
+
+```js
+import * as sst from "@serverless-stack/resources";
+
+flutterrt default class MyStack extends sst.Stack {
```
Comment (fwang) [inline _examples/how-to-create-a-flutter-app-with-serverless.md:69]: `flutterrt` typo?

File: _examples/how-to-create-a-flutter-app-with-serverless.md [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+Our API is powered by a Lambda function. In the function we'll read from our DynamoDB table.
+
+{%change%} Replace `src/lambda.js` with the following.
+
+```js
+import AWS from "aws-sdk";
+
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
+flutterrt async function main() {
```
Comment (fwang) [inline _examples/how-to-create-a-flutter-app-with-serverless.md:134]: typo?

File: _examples/how-to-create-a-flutter-app-with-serverless.md [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+```
+
+You should see a `0` printed out.
+
+## Setting up our flutter app
+
+We are now ready to use the API we just created. Let's use [flutter-cli](https://flutter.dev/docs/get-started/install) to setup our flutter app.
+
+{%change%} Run the following in the project root.
+
```
Comment (fwang) [inline _examples/how-to-create-a-flutter-app-with-serverless.md:211]: Add a step for installing the flutter cli.

Comment (Manitej66) [reply _examples/how-to-create-a-flutter-app-with-serverless.md:211]: flutter doesn't use node, need to be installed manually.

**Pull request closed without merging**